akka-projection-cassandra/src/main/scala/akka/projection/cassandra/javadsl/CassandraProjection.scala

```diff
@@ -72,4 +69,19 @@ object CassandraProjection {
    */
   def initializeOffsetTable(systemProvider: ClassicActorSystemProvider): CompletionStage[Done]
 
+  def withAtLeastOnceRecoveryStrategy(
```

Comment:
this should move to `AtLeastOnceCassandraProjection`. The scaladsl impl is correct.

Reply:
we can also just call it `withRecoveryStrategy` and let the type guide.

Reply:
> this should move to `AtLeastOnceCassandraProjection`. The scaladsl impl is correct.

:+1: 

> we can also just call it `withRecoveryStrategy` and let the type guide.

Good catch.  I did this originally, but at some point during the implementation I made it verbose to avoid conflicts for some reason.  I've reverted to `withRecoveryStrategy`